docs/docs/configuration/overview.md

```diff
@@ -174,7 +176,7 @@ By default, OAuth2 Proxy logs all output to stdout. Logging can be configured to
 
 If logging to a file you can also configure the maximum file size (`--logging-max-size`), age (`--logging-max-age`), max backup logs (`--logging-max-backups`), and if backup logs should be compressed (`--logging-compress`).
 
-There are three different types of logging: standard, authentication, and HTTP requests. These can each be enabled or disabled with `--standard-logging`, `--auth-logging`, and `--request-logging`.
+There are three different types of logging: standard, authentication, session refresh and HTTP requests. These can each be enabled or disabled with `--standard-logging`, `--auth-logging`, `--refresh-logging`, and `--request-logging`.
```

Comment:
I don't think adding to our templated log types is the right direction.

It adds way too much complexity both to contributors in how to log & to new installs to have another type of log format to potentially template.

We are also thinking about how to completely refactor logging in the medium term (people seem to want klog or zap).

I think any refresh logs we do are better handled as a type of Auth logs that are refresh specific (eg `AuthRefresh` as a new status type).